lib/mockbuilder.py

```diff
@@ -57,7 +57,6 @@ def initialize(self):
 
     def build(self, package):
         LOG.info("%s: Starting build process" % package.name)
-        self._prepare(package)
```

Comment:
why this line was removed in this patch? It seems to me it should have been removed in the previous patch